### ISSUE TITLE
[11.0][report_xlsx] Empty error message popup after generating .xlsx file and switching to a different app

### ISSUE THREAD

**Issue by Azratosh** [OCA/reporting-engine#368]
I created my own module that generates xlsx reports using `report_xlsx`. Whenever I generate and save a report, and then go to a different app, an error popup appears that isn't displaying anything: 
![grafik](https://user-images.githubusercontent.com/26200153/73862208-854fe780-483e-11ea-826c-06b1eeaab7d4.png)

The odoo console output isn't showing any related messages either. I have noticed that the same thing happens with the `account_standard_report` module, as it is the only other module that also uses `report_xlsx`. I have thoroughly attempted to debug my module, but I couldn't find anything. Maybe it's related to a controller?

**Comment by pedrobaeza** [comment]
Check examples like https://github.com/OCA/account-financial-reporting/tree/11.0/account_financial_report where there's no error using this module.

Closing as no error on the module, but possible misuse of it.

**Comment by sacostaylara** [comment]
@pedrobaeza I'm trying the module you mentioned (account_financial_report) and I'm experiencing the same error that is described in this issue: an empty popup error appears after exporting an xlsx and then switching to another app. 
This happens on enterprise version.

**Comment by pedrobaeza** [comment]
Then it seems you have something incorrect in your end, as the module is working without problems. You can try it on the runbot link in the module README.

**Comment by sacostaylara** [comment]
> Then it seems you have something incorrect in your end, as the module is working without problems. You can try it on the runbot link in the module README.

No, there is nothing incorrect at my end, I'm testing this on a clean enterprise installation with only these OCA modules installed.
In the runbot you mentioned its using community version so you won't be able to notice the error since its not possible to switch between apps, but the error is still there.

I was able to fix the issue, the reason was that the response of the download xlsx is missing the cookie 'fileToken' and the frontend is expeting this to be able to complete the ajax call.  

Here I created a pull request that fixes this issue: https://github.com/OCA/reporting-engine/pull/414
